kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -2,13 +2,13 @@
 
   <!--TODO: VUE2 UNTESTED -->
   <div>
-    <nav-bar-item v-if="loggedIn" tabindex="0" @click="toggleDropdown" @keyup.enter="toggleDropdown">
+    <nav-bar-item v-if="loggedIn" tabindex="0" @click.native="toggleDropdown" @keyup.enter="toggleDropdown">
```

Comment:
I think this is reasonable but I'm curious if there are other strategies you can think of?